src/Model/ValueObject/Identity/UuidFactory/FixedUuidFactory.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Model\ValueObject\Identity\UuidFactory;
+
+use CultuurNet\UDB3\Model\ValueObject\Identity\Uuid;
+
+class FixedUuidFactory implements UuidFactoryInterface
+{
+    private Uuid $uuid;
+
+    public function __construct(Uuid $uuid)
```

Comment:
I can imagine needing multiple uuid's to be generated.
You could construct this factory with an array of Uuids